enhancements/microshift/dummy-gateway-interface-ovn.md

```diff
@@ -0,0 +1,429 @@
+  │       │                                                      │
+  │       │              ┌────────────┐                          │
+  │       │              │  dummy0    │                          │
+  │       │              └─────┬──────┘                          │
+  │    ┌──┴──────┐             │                                 │
+  │    │routes + │     ┌───────┴───────┐                         │
+  │    │iptables ├─────┤     br-ex     │                         │
+  │    └─────┬───┘     │  10.44.0.1/32 │                         │
+  │          │         └───────┬───────┘                         │
+  │          │                 │                                 │
+  │    ┌─────┼─────────────────┼────────────────────────────┐    │
+  │    │     │                 │       OVN Virtual Topology │    │
+  │    │     │        ┌────────┴────────┐                   │    │
+  │    │     │        │ External Switch │                   │    │
+  │    │     │        └────────┬────────┘                   │    │
+  │    │     │                 │                            │    │
+  │    │     │     ┌───────────┴──────────┐                 │    │
+  │    │     │     │     Gateway Router   │                 │    │
+  │    │     │     └───────────┬──────────┘                 │    │
+  │    │     │                 │                            │    │
+  │    │     │         ┌───────┴────────┐                   │    │
+  │    │     │         │  Join Switch   │                   │    │
+  │    │     │         └───────┬────────┘                   │    │
+  │    │     │                 │                            │    │
+  │    │     │   ┌─────────────┴─────────────┐              │    │
+  │    │     │   │    OVN Cluster Router     │              │    │
+  │    │     │   │        10.244.0.1/24      │              │    │
+  │    │     │   └─────────────┬─────────────┘              │    │
+  │    │     │                 │                            │    │
+  │    │     │        ┌────────┴─────────┐                  │    │
+  │    │     │        │    Node Switch   │                  │    │
+  │    │     │        └────────┬─────────┘                  │    │
+  │    │     │                 │                            │    │
+  │    │  ┌──┴────────────┐    │      ┌──────────────┐      │    │
+  │    │  │ mp0           ├────┴──────┤Pod           │      │    │
+  │    │  │ 10.244.0.2/24 │           │10.244.0.50/24│      │    │
+  │    │  └───────────────┘           └──────────────┘      │    │
+  │    │                                                    │    │
+  │    └────────────────────────────────────────────────────┘    │
+  │                                                              │
+  └──────────────────────────────────────────────────────────────┘
+```
+
+There will be no change to the overlay path taken by the east-west traffic
+between the pods. 
+
+The following north-south traffic paths will not be affected:
+
+* Overlay pod to external
```

Comment:
This traffic path is the same as normal local gw mode since the egress traffic will leave OVN-K virtual topology via mp0. Then it will be masqueraded by the host iptables rule and then sent out from eth0.